manifests/init.pp

```diff
@@ -34,13 +34,33 @@
     require => File[$pl_root],
   }
 
+  $pl_logs_base = '/var/log/puppetlabs'
+  file { "${pl_logs_base}/bolt-server":
+    ensure  => directory,
+    owner   => 'root',
+    group   => 'root',
+    recurse => true,
+    mode    => '0750',
+  }
+
+  file { "${pl_logs_base}/bolt-server/bolt-server.log":
+    ensure  => file,
+    require => File["${pl_logs_base}/bolt-server"],
+  }
+
   file { '/root/.puppetlabs/bolt/bolt-project.yaml':
     ensure  => file,
     content => to_yaml( {
         'modulepath' => [
           '/etc/puppetlabs/code/environments/production/site-modules',
           '/etc/puppetlabs/code/environments/production/modules',
         ],
+        'log'        => {
+          '/var/log/puppetlabs/bolt-server/bolt-server.log' => {
+            'append' => true,
+            'level'  => 'trace',
```

Comment:
Looking at the detail level of the trace log, I think we want to make the bolt log level into a plan parameter, defaulting to `debug` probably. This way, the user can change the log level to `trace` or `info` in case that is needed.

Reply:
It makes sense to me! 💡